Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -78,195 +81,234 @@ Copyright Notice
+   [RRL], the introduction of IPv6 transport, the deployment of DNSSEC,
+   changes in DNSSEC key sizes and preparations to roll the root zone's
+   trust anchor.  Together, even the projects listed in this brief
+   summary imply tremendous operational change, all the more impressive
+   when considered the necessary caution when managing Internet critical
+   infrastructure, and the context of the adjacent administrative
+   changes involved in root zone management and the (relatively
+   speaking) massive increase in the the number of delegations in the
+   root zone itself.
+
+   Aspects of the operational structure of the Root Server System have
+   been described in such documents as [TNO], [ISC-TN-2003-1],
+   [RSSAC001] and [RFC7720].  Such references, considered together,
+   provide sufficient insight into the operations of the system as a
+   whole that it is straightforward to imagine structural changes to the
+   root server system's infrastructure, and to wonder what the
+   operational implications of such changes might be.
+
+   The Yeti DNS Project was conceived in May 2015 to provide a captive,
+   non-production testbed upon which the technical community could
+   propose and run experiments designed to answer these kinds of
+   questions.  Coordination for the project was provided by TSIF, the
+   WIDE Project and the Beijing Internet Institute.  Many volunteers
+   collaborated to build a distributed testbed that at the time of
+   writing includes 25 Yeti root servers with 15 operators and handles
+
+
+
+Song, et al.            Expires December 18, 2017               [Page 3]
+
+Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
+
+
+   experimental traffic from individual volunteers, universities, DNS
+   vendors and distributed measurement networks.
 
-   To benefit Internet development as a whole, the Yeti DNS Project was
-   proposed to build a parallel, experimental, live IPv6 DNS root system
-   to discover the limits of DNS root name service and deliver useful
-   technical output.  Possible research agenda will be explored on this
-   testbed, covering several aspects (but not limited to):
+   By design, the Yeti testbed system serves the root zone published by
+   the IANA with only those structural modifications necessary to ensure
+   that it is able to function usefully in the Yeti testbed system
+   instead of the production Root Server system.  In particular, no
+   delegation for any top-level zone is changed, added or removed from
+   the IANA-published root zone to construct the root zone served by the
+   Yeti testbed system.  In this document, for clarity, we refer to the
+   zone derived from the IANA-published root zone as the Yeti-Root zone.
 
-   o  IPv6-only operation
+   This document presents a summary of the experiments carried out as
+   part of the Yeti DNS Project during its first two years of operation.
 
-   o  DNSSEC key rollover
+2.  Areas of Study
```

Comment:
when we list the areas of study, I think awe shoud let audience know (some where in the document)  which parts we can answer in the expereince of yeti, and which parts are still unkonw ,but worth of identifying it as a issue.